src/opendlv-statsd.cpp

```diff
@@ -26,19 +26,31 @@
 int32_t main(int32_t argc, char **argv) {
     int32_t retCode{1};
     auto commandlineArguments = cluon::getCommandlineArguments(argc, argv);
-    if ( (0 == commandlineArguments.count("cid")) ) {
+    if ( (0 == commandlineArguments.count("cid") || 0 == commandlineArguments.count("field")) ) {
         std::cerr << argv[0] << " joins a running OD4 session to relay selected messages or message fields at a given samplerate to statsd." << std::endl;
-        std::cerr << "Usage:   " << argv[0] << " --cid=<OD4 session> [--verbose]" << std::endl;
+        std::cerr << "Usage:   " << argv[0] << " --cid=<OD4 session> --field=<field name>:<sample rate> [--verbose]" << std::endl;
         std::cerr << "         --cid:    CID of the OD4Session to receive messages" << std::endl;
-        std::cerr << "Example: " << argv[0] << " --cid=111 --verbose" << std::endl;
+        std::cerr << "         --field:    name of the field to sample" << std::endl;
+        std::cerr << "Example: " << argv[0] << " --cid=111 --field=Wgs84:5 --verbose" << std::endl;
     }
     else {
         const bool VERBOSE{commandlineArguments.count("verbose") != 0};
+        
+        const int UDP_PACKET_SIZE = 1462;
```

Comment:
Please use these constants for clarity: https://github.com/chrberger/libcluon/blob/master/libcluon/include/cluon/UDPPacketSizeConstraints.hpp#L16-L20